app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -546,8 +546,8 @@ export default Controller.extend({
       this.cancelComposer(differentDraftContext);
     },
 
-    save() {
-      this.save();
+    save(ignore, event) {
```

Comment:
Linters prefer `_ignored` here, or `_force`

Reply:
our linter did not complain though? not against changing it but seems odd that there were no complaints.